ios/RNLinksdk.m

```diff
@@ -99,14 +99,16 @@ - (void)stopObserving {
     NSString *paymentTokenInput = [RCTConvert NSString:configuration[kRNLinkKitConfigPaymentTokenKey]];
     NSString *oauthStateId = [RCTConvert NSString:configuration[kRNLinkKitConfigOAuthStateIdKey]];
     NSString *institution = [RCTConvert NSString:configuration[kRNLinkKitConfigInstitutionKey]];
+    NSString *publicKey = [RCTConvert NSString:configuration[kRNLinkKitConfigPublicKeyKey]];
 
     PLKConfiguration* linkConfiguration;
     BOOL isUsingLinkToken = [linkTokenInput length] > 0 && [linkTokenInput hasPrefix:kRNLinkKitLinkTokenPrefix];
+    BOOL isUsingPublicKey = [publicKey length] > 0;
 
-    if (isUsingLinkToken) {
-      linkConfiguration = [self getLinkTokenConfiguration: configuration];
-    } else {
+    if (isUsingPublicKey) {
       linkConfiguration = [self getLegacyLinkConfiguration: configuration];
```

Comment:
💅  I suggest to name this `getLinkPublicKeyConfiguration`; could be done in a separate PR.